fms_extras/models/speculator.py

```diff
@@ -281,6 +287,12 @@ def factory(**user_kwargs):
     return factory
 
 
+models.register_model(
+    _architecture_name,
+    "llama.ibm.7b_instruct_lab",
```

Comment:
'llama.ibm.7b_...' seems inconsistent with the above `ibm.7b_instruct_lab`

maybe we should also discuss on slack. Wince we're also talking about adding version information to the variant names, it would be nice to settle on a consistent convention for all of these variant names

Reply:
Agreed